From 47187ba5d63a56bb5894708b3f8685089c775a14 Mon Sep 17 00:00:00 2001
From: Chen Lin Z <lin.z.chen@intel.com>
Date: Tue, 22 Oct 2019 11:04:47 +0800
Subject: [PATCH] WA for boot issues

1. trusty issue on new laptops and PCs
2. reboot issues on old laptops and PCs

Tracked-On:
Signed-off-by: Chen Lin Z <lin.z.chen@intel.com>
---
 caas/mixins.spec | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/caas/mixins.spec b/caas/mixins.spec
index 2e89090..124017e 100755
--- a/caas/mixins.spec
+++ b/caas/mixins.spec
@@ -50,7 +50,7 @@ lights: true
 power: true(power_throttle=true)
 debug-usb-config: true(source_dev=dvcith-0-msc0)
 intel_prop: true
-trusty: true(ref_target=celadon_64)
+trusty: false
 memtrack: true
 avb: true
 health: true
@@ -71,7 +71,7 @@ swap: zram_auto(size=1073741824,swappiness=true,hardware=gordon_peak)
 art-config: true
 psdapp: true
 debugfs: true
-disk-encryption: true
+disk-encryption: false
 factory-scripts: true
 filesystem_config: common
 load_modules: true
-- 
1.9.1

